thirdeye/thirdeye-frontend/app/pods/components/rootcause-table/component.js

```diff
@@ -35,7 +35,12 @@ export default Ember.Component.extend({
   data: Ember.computed(
     'records',
     function () {
-      return Object.values(this.get('records'));
+      let values = Object.values(this.get('records'));
+      values.forEach(value => {
```

Comment:
nit: let's use `parens` around `(data)` since this array function spans across multiple line.